pkg/filters/custom_filter.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package filters
```

Comment:
Wondering whether there's a better name for it. But, maybe just `bloom` or `bloomfilter` since it doesn't contain any other filters. `liveset`, `liveness` also come to mind.

Reply:
`live**` variants doesn't mean that much to me, perhaps something that involves `sn`, `pieces`, otherwise `bloomfilter` looks better.